specificatie/naam.yaml

```diff
@@ -0,0 +1,172 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Aanhef:
+      type: string
+      description: |
+        Kun je gebruiken als aanhef in een brief gericht aan persoon.
+      example: "Hoogwelgeboren heer"
+    Aanschrijfwijze:
+      type: string
+      description: |
+        Samengestelde naam die je kunt gebruiken in de communicatie met de persoon.
+      example: "H.W. baron van den Aedel"
+    GebruikInLopendeTekst:
+      type: string
+      description: |
+        Naam van persoon die je kunt gebruiken als je in lopende tekst (bijvoorbeeld in een brief) aan persoon refereert.
+      example: "baron Van den Aedel"
+    Geslachtsnaam:
+      type: string
+      maxLength: 200
+      description: |
+        De achternaam van een persoon.
+      example: "Vries"
+    RegelVoorafgaandAanAanschrijfwijze:
+      type: string
+      description: |
+        Deze regel moet als aparte regel boven de aanschrijfwijze worden geplaatst. Komt alleen voor bij personen met een adellijke titel of predicaat.
+      example: "De hoogwelgeboren heer"
+    VolledigeNaam:
+      type: string
+      description: |
+        Combinatie van voornamen, voorvoegsels en geslachtsnaam, zonder predikaat, adellijke titel of toevoeging van een eventuele partnernaam.
+      example: "Pieter Jan de Vries"
+    Voorletters: 
+      type: string
+      description: |
+        De voorletters van de persoon, afgeleid van de voornamen.
+      example: "P.J."
+    Voornamen:
+      type: string
+      maxLength: 200
+      description: |
+        De verzameling namen voor de geslachtsnaam, gescheiden door spaties.
+    Voorvoegsel:
+      type: string
+      maxLength: 10
+      example: "de"
+    WaardetabelMetSoort:
+      allOf:
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+        - properties:
+            soort:
```

Comment:
deze component is nu generiek gedefinieerd als "WaardetabelMetSoort". Wordt nu alleen nog gebruikt voor adellijketTtelPredikaat. Daarin zijn maar twee mogelijke waarden: "titel" en "predikaat". In verband met #913 kan je hier een enumeratie van maken. De kans dat die enumeratie uitgebreid wordt (enumeratie onvoldoende evolvable) is nul, want deze zit al in de naam van het gegeven.

Reply:
Vind ik prima, maar dan zou ik de type niet WaardetabelMetSoort noemen. Dit suggereert iets generieks. Voorstel: AdellijkeTitelPredikaatType?

Reply:
voorstel "AdellijkeTitelPredikaatType " is wat mij betreft prima

Reply:
aangepast